cmd/helm/status.go

```diff
@@ -143,6 +143,10 @@ func PrintStatus(out io.Writer, res *services.GetReleaseStatusResponse) {
 	if len(res.Info.Status.Notes) > 0 {
 		fmt.Fprintf(out, "NOTES:\n%s\n", res.Info.Status.Notes)
 	}
+
+	if res.Chart.Metadata.Deprecated {
+		fmt.Printf("WARNING: This chart has been deprecated.\n")
```

Comment:
Shouldn't this be printed to `out` (or stderr)?

Reply:
yes this should. See line 144 for an example.